docs/configuration.md

```diff
@@ -399,6 +399,9 @@ your [`extend-include`](settings.md#extend-include) setting, like so:
 This will prompt Ruff to discover Jupyter Notebook (`.ipynb`) files in any specified
 directories, then lint and format them accordingly.
 
+!!! info
+    Notebooks are linted and formatted by default when using [preview mode](preview).
+
```

Comment:
nit: I think it might be useful for this note to be right below the "Jupyter Notebook Discovery" section